### ISSUE TITLE
A11y_Visual Studio Code_HomePage_Pause Stop Hide: There is no mechanism for a user to pause/stop/hide scrolling content(VS Code for languages) that starts automatically when page is opened.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4528]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#WCAG2.2.2#Pause Stop Hide#Visual Studio Code Services#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Repro Steps:

1. Hit [Visual Studio Code](https://code.visualstudio.com/) link
2. Tab till 'Get Started' button.
3. Verify whether there is any mechanism for a user to pause/stop/hide scrolling content (VS Code for languages) 

### Actual Result:
There is no mechanism for a user to pause/stop/hide scrolling content(VS Code for languages) that starts automatically when page is opened.

### Expected Result:
For any moving or scrolling information that starts automatically, lasts more than five seconds, there should be a mechanism for the user to pause, stop, or hide it. For 'VS Code for languages' scrolling content, there should be a mechanism to pause, stop, or hide it.

### User Impact:
Users with cognitive disabilities cannot grasp the scrolling content if users are not provided enough time to read and use content.

### Attachment:

https://user-images.githubusercontent.com/83570638/117248857-43a55680-ae5e-11eb-90ea-93336efd58db.mp4

**Comment by chrisdias** [comment]
Instead of trying to augment the animation, we can change the content on the page for this section to not be a scrolling list, but instead convey the point that there is support in VS Code for many many technologies.

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed please find the attachment for reference.
URL: https://code.visualstudio.com/
![Bug_4528_Fixed](https://user-images.githubusercontent.com/94966876/218008448-bf4cd5ac-92f3-4a59-8d71-083d2884a693.png)
